format with ruff

nikochiko opened the pull request:
### Q/A checklist

- [x] If you add new dependencies, did you update the lock file?
```bash
poetry lock --no-update
```
- [x] Run tests 
```bash
ulimit -n unlimited && ./scripts/run-tests.sh
```
- [x] Do a self code review of the changes - Read the diff at least twice.  
- [x] Carefully think about the stuff that might break because of this change - this sounds obvious but it's easy to forget to do "Go to references" on each function you're changing and see if it's used in a way you didn't expect. 
- [x] The relevant pages still run when you press submit
- [x] The API for those pages still work (API tab)
- [x] The public API interface doesn't change if you didn't want it to (check API tab > docs page)
- [x] Do your UI changes (if applicable) look acceptable on mobile?
- [x] Ensure you have not regressed the import time unless you have a good reason to do so. 
You can visualize this using tuna:
```bash
python3 -X importtime -c 'import server' 2> out.log && tuna out.log
```
To measure import time for a specific library:
```bash
$ time python -c 'import pandas'

________________________________________________________
Executed in    1.15 secs    fish           external
   usr time    2.22 secs   86.00 micros    2.22 secs
   sys time    0.72 secs  613.00 micros    0.72 secs
```
To reduce import times, import libraries that take a long time inside the functions that use them instead of at the top of the file:
```python
def my_function():
    import pandas as pd
    ...
```

### Legal Boilerplate

Look, I get it. The entity doing business as “Gooey.AI” and/or “Dara.network” was incorporated in the State of Delaware in 2020 as Dara Network Inc. and is gonna need some rights from me in order to utilize my contributions in this PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Dara Network Inc can use, modify, copy, and redistribute my contributions, under its choice of terms.

devxpy commented:
Thanks much needed but I'm going to hold this off until a good point where other PRs are merged!

devxpy commented:
go now

nikochiko commented:
so the full package (linter + formatter) makes more changes. the earlier changes were only linter.

please take a look at these are some code changes i made to make linter pass: https://github.com/GooeyAI/gooey-server/pull/637/files/587a7fc681f9b02aa55f9d29639d173c52a2a5ed..4c84ae67fb908d400170ffdb463f15306c31ed59
- removed `_reflect(...)` and `_add_shadow(...)` from image segmentation because they weren't used anywhere in the code
- removed `page = VideoBotsStatsPage()` in `bots/tests.py` because it was unused. i can revert if this was intentional and part of the tests
- removing other unused variables

more changes are due to the formatter. i added so we have consistent styles - and ignored some rules to match our codebase.
only the linter is also good and does most of what we needed `black` for. i don't mind a few inconsistencies in style if we don't use the formatter.

nikochiko commented on handles/models.py:
```diff
@@ -246,7 +245,7 @@ def _generate_handle_options_for_team_workspace(
     yield handle_name[:HANDLE_MAX_LENGTH]
     for i in range(1, 10):
-        yield f"{handle_name[:HANDLE_MAX_LENGTH-1]}{i}"
+        yield f"{handle_name[: HANDLE_MAX_LENGTH - 1]}{i}"
```
this is the only change where i don't like ruff's suggestion

nikochiko commented:
note: https://github.com/GooeyAI/gooey-server/pull/640 needs to be merged before this PR because `ModelTrainer` right now depends on one of black's dependencies :p 

the tests still pass because i think github's default python image comes with black & its dependencies.

`poetry run python -X importtime 'server'` failed locally is when i realised
